src/claps/claps.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Post, Param, UseGuards, Request } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import {
+  ApiBearerAuth,
+  ApiTags,
+  ApiParam,
+  ApiCreatedResponse,
+} from '@nestjs/swagger';
+
+import { Article } from '@src/articles/domain/article';
+import { ClapsService } from '@src/claps/claps.service';
+@ApiTags('Claps')
+@Controller({
+  path: 'articles',
+  version: '1',
+})
+export class ClapsController {
+  constructor(private readonly clapsService: ClapsService) {}
+
+  @ApiBearerAuth()
+  @UseGuards(AuthGuard('jwt'))
+  @Post(':id/clap')
+  @ApiParam({
+    name: 'id',
+    type: String,
+    required: true,
+  })
+  @ApiCreatedResponse({
+    type: Article,
+  })
+  async clapArticle(@Param('id') id: string, @Request() request) {
```

Comment:
async keyword can be removed, there is no await in the method.

Reply:
Fixed